guides-markdown/wallets.md

```diff
@@ -218,9 +237,36 @@ We can create a second account and name it after a customer.
 	const index = 0; //index of 0 means the first key among the receiving addresses
 	const jdPrivateKey0 = jdExtendedPrivateKey.derive(branch).derive(index);
 	const jdWalletKey0 = WalletKey.fromHD(jdAccount, jdPrivateKey0, branch, index);
-	//the private key below can be imported into almost any bitcoin wallet, HD or non-HD, and used to spend bitcoins from the corresponding address
+	//the private key below can be imported into almost any bitcoin wallet, HD or non-HD, and used to spend coins from the corresponding address
 	console.log(jdWalletKey0.getPrivateKey('base58')); 
 	//cNZfR3NhQ9oCP3pTjvPZETUuTWZo2k6EXtfczvbWyv7FdjMhppvJ
+```
+
+#### Generating Mnemonics and Recovering Keys
+
+Finally, you can create mnemonics manually and seed new wallets with them. And if you need to generate keys from a mnemonic you provide - either by recreating a wallet or by extracting specific keys - you can do that as well.
+
+```javascript
+	//can we generate a mnemonic with twice as many bits of entropy to future proof against brute force attacks from the next millenium? sure we can.
+	const mnemonic24 = new Mnemonic({bits: 256});
+	console.log(mnemonic24.toString());
+	//page unknown ladder thunder airport merry run ball inject clinic danger valley equip consider normal twist casual duck essay almost trade regular two segment
+
+	//what if we need to recover the wallet we've created above? no problem.
+	const mnemonic = new Mnemonic('again choose noble warrior print thrive post glare movie glove animal legal');
+	const masterKey = HD.fromMnemonic(mnemonic);
+	//this wallet will generate all the same accounts, keys, addresses, and tokens if swapped in for the 'wallet' variable in examples above
+	const recoveredWallet = await wdb.create({master: masterKey});
+	
+	//we can also recover only the keys for John Doe's first receiving address without recreating the wallet or account
+	//this time we'll skip instantiating the account and use the BIP44 path for the second account, first branch, first index
+	const jdRecoveredPrivateKey = masterKey.derivePath("m/44'/1'/1'/0/0");
```

Comment:
I think the path or the path description needs to be corrected. The first number (after 44) indicates the network, so this would be "BIP44 path for the second account, first address on receiving branch on the Bitcoin Testnet". This isn't wrong, it just might be good to be explicit.